packages/react-components/src/types/ParticipantListParticipant.ts

```diff
@@ -32,6 +35,23 @@ export type RaisedHand = {
   raisedHandOrderPosition: number;
 };
 
+/* @conditional-compile-remove(reaction) */
+/**
+ * Reaction state with reaction type to render
+ *
+ * @public
+ */
+export type Reaction = {
+  /**
+   * Specifies the type of reaction videoTile should render i.e. like, heart etc.
+   */
+  reactionType: string;

```

Comment:
instead of `string` - should we list what we support?
```suggestion
  reactionType: 'like' | 'heart' | ' ...
```

Reply:
I guess this will be determined by the solution for how we retrieve the assets

Reply:
Yes. also how we want to use capability. For now, can we keep it as string?